app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
```

Comment:
For future please, don't use `* \` as a line break , you can add W504 flake8 rule to ignored ones or create a few temporal variables